server/src/frontend_api.rs

```diff
@@ -11,8 +11,21 @@ use unleash_yggdrasil::{Context, EngineState};
 
 use crate::types::{EdgeJsonResult, EdgeSource, EdgeToken};
 
+///
```

Comment:
Extra line perhaps?

```suggestion

```

Reply:
Yeah, but we should add a proper comment here, it will be parsed and added to the description of the call.